src/json/members/choiyoon.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Yoonjung Choi",
+    "pennkey": "choiyoon",
+    "major": "Undecided",
+    "school": "SAS",
+    "bio": "Hi, this is Yoonjung. I'm a BizDev for Penn Clubs.",
+    "hometown": "Hong Kong",
+    "team": "Penn Clubs",
+    "roles": ["Business Developer"],
+    "photo": "https://media.licdn.com/dms/image/D5603AQHYyI1WDPwFvw/profile-displayphoto-shrink_400_400/0/1678278503421?e=1701302400&v=beta&t=rGoJZl7lZCxJwDz3xJPJnMk4IbHkzY85eyXdMnB1mZ8",
```

Comment:
It's easy for this type of photo to no longer have a valid URL after a certain period of time. Rather than right clicking on LinkedIn profile to copy image URL, I would recommend using a hosting site like Imgur.